### PR TITLE
API integration for each game type

### PR DESCRIPTION
**Pull request by yangwonjoon**
[setting]
- proxy for gameType api
- prettier for tab width
- context api for gameType

[update]

- Remove unnecessary files
- add question for each gameType

[error]
"can't automatically merge"  clear

Error Reference
https://stackoverflow.com/questions/31973318/github-cant-automatically-merge
https://gollumnima.github.io/posts/wecode9_7TIL_git_error

### CODE REVIEW

File: src/components/common/Logo.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import logo from '../../assets/images/logo.svg';
+
+interface LogoProps {
+  gameType: string;
+}
+
+export const Logo = ({ gameType }: LogoProps) => {
+  const getTitle = (type: string) => {
```
Comment (kim-song-jun) [inline src/components/common/Logo.tsx:8]: use object instead of switch

Comment (kim-song-jun) [reply src/components/common/Logo.tsx:8]:
```js
const title = {
   'love' : 'Love Tarot'
}

return title[key]
```

File: src/components/common/ThreeCardSelect.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+              <Button
+                className="cursor-default bg-main m-3 hover:bg-main-darker"
+                onClick={close}
+              >
+                Retry
+              </Button>
+            )}
+            <Button
+              className="cursor-default bg-main hover:bg-main-darker"
+              onClick={() => navigate('/main')}
```
Comment (kim-song-jun) [inline src/components/common/ThreeCardSelect.tsx:50]:
```js
const handleClickSomeButton = () => {
   navigate('/main')
}
```

File: src/hooks/useCardSpread.ts [hunk 3]
```diff
@@ -1,44 +1,59 @@
...
-
-    const spread = (): ISpreadCardPosition[] => Array.from({ length: 12 }, (_, i) => {
-        const angle = (180 / 11 * i) * (Math.PI / 180);
-        return { x: 600 * Math.cos(angle), y: 200 * Math.sin(angle) };
+export const useCardSpread = (): IUseCardSpreadReturn => {
+  const [isAnimated, setIsAnimated] = useState<boolean>(false);
+  const [positions, setPositions] = useState<ISpreadCardPosition[]>([]);
+  const [selectedCards, setSelectedCards] = useState<number[]>([]);
+  const [visibleCards, setVisibleCards] = useState<boolean[]>(
+    Array(12).fill(true)
```
Comment (kim-song-jun) [inline src/hooks/useCardSpread.ts:11]: Do not use magic number

**Review by kim-song-jun — COMMENTED** [review]
LGTM

File: src/components/common/Logo.tsx (hunk 1 above)
Comment (yangwonjoon) [reply src/components/common/Logo.tsx:8]: complete

File: src/components/common/ThreeCardSelect.tsx (hunk 2 above)
Comment (yangwonjoon) [reply src/components/common/ThreeCardSelect.tsx:50]: complete

File: src/hooks/useCardSpread.ts (hunk 3 above)
Comment (yangwonjoon) [reply src/hooks/useCardSpread.ts:11]: complete

**Pull request merged**